armi/interfaces.py

```diff
@@ -539,6 +647,42 @@ def apply(self, reactor):
         raise NotImplementedError()
 
 
+def _setTightCouplerByInterfaceFunction(klass, cs):
+    """
+    Return an instance of a ``TightCoupler`` class or ``None``.
+
+    Parameters
+    ----------
+    klass : Interface
+        Interface that a ``TightCoupler`` object will be added to.
+
+    cs : Settings
+        Case settings that are parsed to determine if tight coupling is enabled
+        globally and if both a target parameter and convergence criteria defined.
+    """
+    # No type coupling if there is no function for the Interface defined.
+    if klass.function is None:
```

Comment:
As is pointed out in the existing TODO statement [here](https://github.com/terrapower/armi/blob/9d6a405f10a0084b2fea8721f2b6d08c16a0b7a8/armi/interfaces.py#L111-L117) `*.function` is not a great name. Is it too far out of scope in this PR to change it? Maybe to `interfaceAction` or something like that?

Reply:
Yes outside the scope.

Reply:
https://github.com/terrapower/armi/issues/1068